caravel/utils.py

```diff
@@ -202,6 +202,8 @@ def init(caravel):
     for perm in perms:
         if perm.permission.name == 'datasource_access':
             continue
+        if perm.permission.name == 'database_access':
```

Comment:
NIT: could go with in if `perm.permission.name in ('datasource_access', 'database_access'):` and get rid of 205-206